### PR TITLE
Fix DG bugs

### PR DESCRIPTION
**Pull request by peironggg**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -119,9 +118,9 @@ The `UI` component,
 1. The result of the command execution is encapsulated as a `CommandResult` object which is passed back to the `Ui`.
 1. In addition, the `CommandResult` object can also instruct the `Ui` to perform certain actions, such as displaying help to the user.
 
-Given below is the Sequence Diagram for interactions within the `Logic` component for the `execute("delete 1")` API call.
+Given below is the Sequence Diagram for interactions within the `Logic` component for the `execute("delete -c 1")` API call.
 
-![Interactions Inside the Logic Component for the `delete -c 1` Command](images/DeleteSequenceDiagram.png)
+![Interactions Inside the Logic Component for the `delete -c 1` Command](images/DeleteConsultationSequenceDiagram.png)
```
Comment (ngzhenteng) [inline docs/DeveloperGuide.md:123]: Great catch!

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -190,7 +183,7 @@ In this section we will explain how the `Automatic Tab Switching Feature` featur
  
 #### What is Automatic Tab Switching
 `Automatic Tab Switching` is a feature where the displayed tab automatically changes to the relevant tab: `Student
-` `Missions` `Quests` `Consultations` `Mastery Checks` and `Tasks`, for the user's input command.
+` `Missions`, `Quests`, `Consultations`, `Mastery Checks` and `Tasks`, for the user's input command.
```
Comment (ngzhenteng) [inline docs/DeveloperGuide.md:186]: Thank you for the impeccable attention to detail.

**Review by ngzhenteng — COMMENTED** [review]
LGTM, thank you for the quick fixes!

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (Asuraxsoul) [reply docs/DeveloperGuide.md:186]: Impeccable attention to detail 👍

Comment (wilinetan) [reply docs/DeveloperGuide.md:186]: Indeed so 👍

**Pull request merged**